deploy/deploy_staking.ts

```diff
@@ -0,0 +1,92 @@
+import { ethers } from 'hardhat';
+import { HardhatRuntimeEnvironment } from 'hardhat/types';
+import { DeployFunction } from 'hardhat-deploy/types';
+import 'hardhat-deploy-ethers';
+import hre from 'hardhat';
+
+import * as addresses from '../deployed.json';
+import { ContractReceipt, ContractTransaction } from 'ethers';
+
+const deployTo = process.env.DEPLOYTO;
+const networkID = 5; //test network
```

Comment:
The network ID should be given as a parameter to this script since we'll deploy to several networks.